src/components/base/SideBar.vue

```diff
@@ -0,0 +1,192 @@
+      </p>
+      <ul class="menu-list">
+        <li>
+          <router-link to="/">
+            <i class="fas fa-home mr-1" />
+            <span v-if="!collapsed">Home</span>
+          </router-link>
+        </li>
+        <li>
+          <router-link to="/about">
+            <i class="fas fa-info-circle mr-1" />
+            <span v-if="!collapsed">About</span>
+          </router-link>
+        </li>
+      </ul>
+      <p v-if="!collapsed" class="menu-label">Exploration Datasets</p>
+      <p v-else class="menu-label">
+        <abbr title="Exploration Datasets">DATA</abbr>
+      </p>
+      <ul class="menu-list">
+        <li v-for="item in DATASETS" :key="item.route">
+          <router-link
+            v-if="item.available"
+            :class="{ 'is-active': activeRoute === item.route }"
+            :aria-disabled="!item.available"
+            :to="'/dataset/' + item.route"
+          >
+            <span v-if="!collapsed">{{ item.name }}</span>
+            <span v-else
+              ><abbr :title="item.name" class="collapsed-flex-item">
+                <i class="fas fa-poll mr-1" />
+                <span>{{ initials(item.name) }}</span>
+              </abbr></span
+            >
+          </router-link>
+          <a
+            v-else
+            disabled
+            :aria-disabled="!item.available"
+            class="not-allowed-cursor"
+          >
+            <span v-if="!collapsed">{{ item.name }}</span>
+            <span v-else
+              ><abbr :title="item.name" class="collapsed-flex-item">
+                <i class="fas fa-poll mr-1" />
+                <span>{{ initials(item.name) }}</span>
+              </abbr></span
+            >
+            <span v-if="!collapsed" class="tag is-light is-warning ml-2"
+              >coming soon</span
+            ></a
+          >
+        </li>
+      </ul>
+      <p v-if="!collapsed" class="menu-label">Resources</p>
+      <p v-else class="menu-label">
+        <abbr title="Resources">Src</abbr>
+      </p>
+      <ul class="menu-list">
+        <li v-for="(item, i) in RESOURCES" :key="i">
+          <a :href="item.link">
+            <span v-if="!collapsed">
+              <i class="fas mr-1" :class="item.icon" />
+              {{ item.title }}
+            </span>
+            <span v-else>
+              <abbr :title="item.title" class="collapsed-flex-item">
+                <i class="fas mr-1" :class="item.icon" />
+                {{ item.initials }}
+              </abbr>
+            </span>
+          </a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+
+const emit = defineEmits(["toggle"]);
+
+const initials = (str) => {
+  return str
+    .split(" ")
+    .map((s) => s[0].toUpperCase())
+    .join("");
+};
+
+const collapsed = ref(false);
+
+const DATASETS = [
+  {
+    name: "RI Buoy Data",
+    route: "ri-buoy",
+    available: true,
```

Comment:
Is there an instance where any of these datasets aren't available?

Reply:
historically, we've listed things as coming soon when we start working on them, but don't have them yet.  Currently, all are available

Reply:
Ah okay! That makes sense